src/Domain/Entities/Products/ValueObjects/ProductTitleValidator.cs

```diff
@@ -0,0 +1,20 @@
+﻿using FluentValidation;
+
+namespace CleanCompanyName.DDDMicroservice.Domain.Entities.Products.ValueObjects;
+
+internal class ProductTitleValidator : AbstractValidator<ProductTitle>
+{
+    private const int MinLength = 5;
+    private const int MaxLength = 350;
+
+    public ProductTitleValidator()
+    {
+        RuleFor(productTitle => productTitle.Value)
+            .NotEmpty()
+                .WithErrorCode("Product_Title_NotEmpty")
```

Comment:
The error code, I guess, is correct; but if it would be a message it has to be the opposite as: `.NotEmpty` checks that the property is not null, empty or whitespace. On the other hand, the `.Empty` checks if the property has a value

If we are using error codes as messages, it should be updated to `Product_Title_Empty` 😆

Reply:
I guess that depends on the point of view.
But if we want to have error codes auto-generated the convention needs to follow this way. Otherwise, the proposed solution won't make sense.

Reply:
We can always discuss forward in the initial PR or move this discussion to the discussions part

Reply:
Yeah, I prefer to stick to the conventions, just pointing that in this case the error code means the contrary, just we have to be aware of that :)